client/my-sites/invites/invite-accept-logged-in/index.jsx

```diff
@@ -153,4 +153,9 @@ class InviteAcceptLoggedIn extends Component {
 	}
 }
 
-export default connect( null, { acceptInvite } )( localize( InviteAcceptLoggedIn ) );
+export default connect(
+	( state ) => ( {
+		emailVerificationSecret: getCurrentQueryArguments( state ).email_verification_secret,
```

Comment:
`getCurrentQueryArguments` can return `null`

```suggestion
		emailVerificationSecret: getCurrentQueryArguments( state )?.email_verification_secret,
```